[Docs]  Add setup content from GET (#1770)
grafana/tempo: docs/tempo/website/getting-started/example-demo-app.md

# Example setups

> These examples show various deployment and [configuration]({{< relref "../configuration" >}}) options.
> They include trace generators so an existing application is not necessary to get started experimenting with Tempo. 
> 
> Refer to [Set up a Tempo cluster]({{< relref "../setup">}}) for instructions on setting up basic Tempo clusters and testing data.
> 
> If you are interested in instrumentation, check out these [examples]({{< relref "./instrumentation" >}}).

Reviewer: ```suggestion
The following examples show various deployment and configuration options using trace generators so you can get started experimenting with Tempo without an existing application.

For more information on Tempo setup and configuration see: 
* [Set up a Tempo cluster]({{< relref "../setup">}})
* [Tempo configuration]({{< relref "../configuration" >}}) 

If you are interested in instrumentation, see [Tempo instrumentation]({{< relref "../instrumentation" >}}).
```

Author: Thank you for the suggestions.

# Example setups

The following examples show various deployment and configuration options using trace generators so you can get started experimenting with Tempo without an existing application.

> For more information on Tempo setup and configuration see: 

Reviewer 2: ```suggestion
For more information about Tempo setup and configuration, see: 
```
"on" means "about" here

Author: Done.

## Tanka

> The Jsonnet based [example](https://github.com/grafana/tempo/tree/main/example/tk) shows a complete microservice based deployment.

Reviewer: ```suggestion
To view an example of a complete microservice-based deployment, see this [Jsonnet based example](https://github.com/grafana/tempo/tree/main/example/tk) shows a complete microservice based deployment.
```

There are monolithic mode and microservices examples.

> Refer to [Deploy on Kubernetes with Tanka]({{< relref "../setup/tanka" >}}) for instructions on setting up a Tempo cluster. 

Reviewer: ```suggestion
To learn how to set up a Tempo cluster, see [Deploy on Kubernetes with Tanka]({{< relref "../setup/tanka" >}}).
```
